engine/src/test/java/net/jqwik/engine/TestDescriptorBuilder.java

```diff
@@ -91,7 +91,7 @@ public TestDescriptor build(TestDescriptor parent) {
 
 	private TestDescriptor createDescriptor(TestDescriptor parent) {
 		if (element instanceof JqwikTestEngine)
-			return new JqwikEngineDescriptor(engineId());
+			return new JqwikEngineDescriptor(engineId(), ((JqwikTestEngine)element).buildConfiguration());
```

Comment:
@jlink re: https://github.com/jlink/jqwik/pull/145/files#r548860845

`buildConfiguration` wasn't private because I was calling it here (in order to propagate the configuration). I admittedly didn't dig into how jqwik's tests use itself to run. I've modified this to pass in `null`, as all the tests continue to pass.